plugins/legacy-preset-chart-nvd3/src/Area/controlPanel.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { t } from '@superset-ui/translation';
+import { sections } from '@superset-ui/control-utils';
+import {
+  lineInterpolation,
+  showBrush,
+  showLegend,
+  showControls,
+  xAxisLabel,
+  bottomMargin,
+  xTicksLayout,
+  xAxisFormat,
+  yLogScale,
+  yAxisBounds,
+  xAxisShowMinmax,
+  richTooltip,
+  timeSeriesSection,
+} from '../NVD3Controls';
+
+export default {
+  requiresTime: true,
+  controlPanelSections: [
+    timeSeriesSection[0],
+    {
+      label: t('Chart Options'),
+      expanded: true,
+      controlSetRows: [
+        [showBrush, showLegend],
+        [
+          lineInterpolation,
+          {
+            name: 'stacked_style',
+            config: {
+              type: 'SelectControl',
+              label: t('Stacked Style'),
+              renderTrigger: true,
+              choices: [
+                ['stack', 'stack'],
+                ['stream', 'stream'],
+                ['expand', 'expand'],
+              ],
+              default: 'stack',
+              description: '',
+            },
+          },
+        ],
+        ['color_scheme', 'label_colors'],
+        [richTooltip, showControls],
+      ],
+    },
+    {
+      label: t('X Axis'),
+      expanded: true,
+      controlSetRows: [
+        [xAxisLabel, bottomMargin],
+        [xTicksLayout, xAxisFormat],
+        [xAxisShowMinmax, null],
+      ],
+    },
+    {
+      label: t('Y Axis'),
+      expanded: true,
+      controlSetRows: [
+        ['y_axis_format', yAxisBounds],
+        [yLogScale, null],
+      ],
+    },
+    timeSeriesSection[1],
+    sections.annotations,
+  ],
+  controlOverrides: {
```

Comment:
Could we apply these overrides in-place instead of having this section here?

```
['color_scheme', 'label_colors']
```

```
[
  'color_scheme', // replace with the object and set renderTrigger to false
  'label_colors'
]
```

Reply:
I think that's a good idea, but for now I'm just trying to move the control panels while keeping them as intact as possible to minimize the chance of breaking things.

Reply:
@kristw I'm making an internal JIRA ticket to track this, and we can go through and audit and remove (where possible) all `controlOverrides`. If you think it's safe enough, we can then consider removing the code that enables them, as well. We'll discuss further with you :)